frontend/pages/hosts/details/cards/HostHeader/_styles.scss

```
@@ -12,7 +12,8 @@
     background-color: $ui-warning;
     padding: $pad-xsmall;
     font-size: $xx-small;
-    // Surface stays yellow in both themes; text must stay dark for contrast.
+    // Warning + error surfaces shift between themes; text stays dark so the
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)